platform.txt

```diff
@@ -85,6 +85,16 @@ compiler.size.cmd=xtensa-lx106-elf-size
 # This can be overridden in boards.txt
 build.extra_flags=-DESP8266
 
+# Pre and post build hooks
+build.opt.name=build_opt.h
+build.opt.path={build.path}/sketch/{build.opt.name}
+
+# extras.path={build.core.path}/extras
+extras.path={runtime.platform.path}/tools/extras
```

Comment:
Can we just place prebuild.py in the tools directory and avoid yet another subdir with a single file, please?